xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
I agree, I think explaining that `__set__` isn't a sensible operation on a field which represents an entire filesystem would be reasonable.